src/App.tsx

```diff
@@ -9,24 +9,22 @@ interface Todo {
 function App() {
   const [items, setItems] = useState<Todo[]>([]);
 
+  const todos = items.reduce((acc: any[], obj) => {
```

Comment:
Add the right type. "any" should not be used :)

Reply:
Done